index.html

```diff
@@ -0,0 +1,195 @@
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/chef-icon.png" alt="featureicon" />
+              <h3>Chefs</h3>
+            </div>
+            <p>
+              we have well trained chefs that are very professional both cooking
+              and serving.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/dish-plate.png" alt="featureicon" />
+              <h3>Customer service</h3>
+            </div>
+            <p>we have a well relateable customer services.</p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/kitchenware_icon.png" alt="featureicon" />
+              <h3>Equipment</h3>
+            </div>
+            <p>
+              our cooking materials are well structured and gotten from the
+              best. And also a neat working environment.
+            </p>
+          </div>
+
+          <div class="activity flex">
+            <div class="flex imgiconwrapper">
+              <img src="image/cook-pot.png" alt="featureicon" />
+              <h3>Feedback</h3>
+            </div>
+            <p>we've had a lot of good reports from our costumers.</p>
+          </div>
+        </div>
+
+        <button type="button">See other feedback</button>
+        <a href="#" class="desktop-mainprogramBtn">place your order</a>
+      </section>
+    </main>
+
+    <section class="section-menu flex">
+      <div class="flex menuHeader">
+        <h3>our Menu</h3>
+        <hr />
+      </div>
+
+      <div class="menus-wrapper flex"></div>
+      <button type="button" id="moreBtn" class="menu-button">
+        MORE <img src="image/dropdown.png" alt="dropdown arrow" />
+      </button>
+    </section>
+
+    <footer>
+      <div class="footerHeader flex">
+        <h2>Partners</h2>
+        <hr />
+      </div>
+      <div class="partnerIcons">
+        <div class="flex imgone">
+          <img
+            class="partnerone"
+            src="image/valentine.png"
+            alt="valentineEatry Logo"
+          />
+        </div>
+        <div class="flex imgone">
+          <img
+            class="partnertwo"
+            src="stewardfood.png"
+            alt="stewardfood Logo"
+          />
+        </div>
+        <div class="flex imgone">
+          <img
+            class="partnerthree"
+            src="image/momenteatry.png"
+            alt="momenteatry Logo"
+          />
+        </div>
+      </div>
+
+      <div class="Footerlogo flex">
+        <a class="logoimage" href="./index.html"
+          ><img src="image/glass_red_icon1.png" alt="rejovankitchen Logo"
+        /></a>
+        <div class="copyrights">
+          <p>Created by iconfinder Website</p>
+          <p>Original Design by Cindy Shin</p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Kindly add 2 more partners in the partner section.
- [ ] Please fix the broken link in the second image.
- [ ] ### OPTIONAL-- You can find another image for the glass image. It doesn't look clear.
- [ ] After adding to the image don't forget in the mobile version partners layout change.
- [ ] Please don't forget to do the same thing on about.html page.




![image](https://user-images.githubusercontent.com/38635158/175648221-494d7d87-7118-46d0-889c-d0d2662a7d61.png)

___
### Original
![image](https://user-images.githubusercontent.com/38635158/175648291-f4dfdb0a-dbf7-4f6a-89af-d7fdd74d1e5b.png)

___
### Original mobile version
![image](https://user-images.githubusercontent.com/38635158/175648778-1d8322cc-840a-48a8-ba8b-5c5268cda763.png)